docs/api/crash-reporter.md

```diff
@@ -27,6 +27,10 @@ crashReporter.start({
     * Only string properties are send correctly.
     * Nested objects are not supported.
 
+**Note:** On OS X, electron uses a new `crashpad` client, which is different
+with the `breakpad` on Windows and Linux. To enable crash collection feature,
+you are required to call `crashReporter.start` API to initiliaze `crashpad` in Browser Process, even you only collect crash report in Renderer Process.
```

Comment:
> The crashpad_handler process need to launch before the renderer process starts, otherwise it will not catch the crash report of the renderer process. In other words, the crashpad initialized work need to be done before the BrowserWindow loads urls.

:+1: 

Currently we prefer "main process" to "browser process" in the documents, and it should be lowercased: https://github.com/atom/electron/issues/1287.